rs/sns/governance/CHANGELOG.md

```diff
@@ -10,6 +10,42 @@ here were moved from the adjacent `unreleased_changelog.md` file.
 
 INSERT NEW RELEASES HERE
 
+
+# 2025-02-03: Proposal 135067
+
+http://dashboard.internetcomputer.org/proposals/135067
+
+## Added
+
+* Enable SNSs to opt in for
```

Comment:
This is released Jan 27

Reply:
double check on the proposals in the dashboard for all of these please

Reply:
I think these are straightened out now